proto/provisioner.proto

```diff
@@ -3,6 +3,7 @@ package provisioner;
 
 service Provisioner {
   rpc ProvisionDatabase(DatabaseRequest) returns (DatabaseResponse);
+  rpc ProvisionArbitraryContainer(ContainerRequest) returns (ContainerResponse);
```

Comment:
I'm not sure we want to add this method to provisioner :thinking: I get that we want something generic, but this seems to limit the generic options to containers.

Maybe we should just have a `Provision` method with a project name, type (string), details (json), and env? And just have it return json?

Reply:
I don't think this endpoint needs to be future proof. If we are adding a more generic way of doing this in the future, it would very likely be a breaking change anyways and a new endpoint would suit that. Since this is only used in local runs, the breaking change would not affect deployers.

Not in favour of returning JSON inside protobufs yet since we have limited knowledge of how future use cases look like. I'm in favour of solving this for more generic cases when we have generic cases to solve this for and the current impl is not enough.

Reply:
I did not think of this being scoped to local runs. Fair enough! :+1: